x-pack/solutions/security/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_flyout.tsx

```diff
@@ -198,6 +199,12 @@ export const ArtifactFlyout = memo<ArtifactFlyoutProps>(
       },
     } = useKibana().services;
 
+    const { euiTheme } = useEuiTheme();
+    const maskProps = useMemo(
+      () => ({ style: `z-index: ${(euiTheme.levels.flyout as number) + 4}` }), // we need this flyout to be above the timeline flyout (which has a z-index of 1003)
```

Comment:
Nit: a little hacky and I know we don't have a better solution for this yet, but it'd be nice to have a variable containing the z-index for the timeline so that we can do something like `'z-index': timeline_zindex + 1`, becase it's not really clear why we are using `4` here 🤔